socorro/cron/jobs/clean_missing_symbols.py

```diff
@@ -0,0 +1,35 @@
+from configman import Namespace
+from crontabber.base import BaseCronApp
+from crontabber.mixins import (
+    with_postgres_transactions,
+    with_single_postgres_transaction,
+)
+
+
+@with_postgres_transactions()
+@with_single_postgres_transaction()
+class CleanMissingSymbolsCronApp(BaseCronApp):
+    app_name = 'clean-missing-symbols'
+    app_description = """
+    See https://bugzilla.mozilla.org/show_bug.cgi?id=1278498
+
+    By default we truncate to the last 7 days. Normally our general
```

Comment:
This says 7 days but the config option's default is 5. Which one should it really be?